src/lib.rs

```diff
@@ -406,6 +406,15 @@ impl fmt::Display for Uuid {
     }
 }
 
+/// Resets the internal dataflow graph.
```

Comment:
Can you please add a more descriptive comment to this method? (e.g., resets seed and creates a new graph...)